setup.cfg

```diff
@@ -18,19 +18,17 @@ filterwarnings =
     ignore:::csp.utils
 
 [flake8]
-ignore = F999,F405
+ignore = F999,F405,W504
```

Comment:
added `W504` to suppress "line break after binary operator" [[e.g.](https://travis-ci.org/mozilla/addons-server/jobs/449769967#L1787)] because I'm not convinced about it.  If we _do_ want it, then lets file a follow-up code-quality issue because there are 520 fails with it removed and it looks like they'll need to be manual changes.

Reply:
Can you file a new issue for this ? I thought we were already obeying this rule. We should look into fixing that properly.

Reply:
https://github.com/mozilla/addons-server/issues/9885